app/src/org/commcare/location/CommCareFusedLocationController.kt

```diff
@@ -29,7 +30,7 @@ class CommCareFusedLocationController(private var mContext: Context?,
     }
 
     private fun requestUpdates() {
-        if (isLocationPermissionGranted(mContext!!)) {
+        if (isLocationPermissionGranted(mContext)) {
             mFusedLocationClient.requestLocationUpdates(mLocationRequest, mLocationCallback, null)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/efa4e6d4807709ea34abb41e48e4b6bd8249726c/app/src/org/commcare/location/CommCareFusedLocationController.kt#L34"></span>Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with <code>checkPermission</code>) or explicitly handle a potential <code>SecurityException</code></td>
    </tr>
  </tbody>
</table>